tests/base.py

```diff
@@ -343,6 +344,11 @@ def align_date_type(self, record, key, value):
             iso_date = dt.strftime(raw_date,  "%Y-%m-%dT%H:%M:%S.%fZ")
             record[key] = iso_date
 
+    def align_number_type(self, record, key, value):
```

Comment:
Not a big deal, but why call the parent method "modify_expected_record" but the specific record modifiers "align_<blank>_type"? shouldn't both be modify or align?